packages/ckeditor5-find-and-replace/src/ui/findandreplaceformview.js

```diff
@@ -417,34 +418,15 @@ export default class FindAndReplaceFormView extends View {
 	 * @returns {module:ui/view~View} The checkbox view instance.
 	 */
 	_createCheckbox( checkboxId, label ) {
-		const checkboxView = new View();
+		const checkboxView = new CheckboxView( this.locale );
 
-		checkboxView.setTemplate( {
-			tag: 'div',
-			attributes: {
-				class: 'ck-find-checkboxes__box'
-			},
-			children: [
-				{
-					tag: 'input',
-					attributes: {
-						type: 'checkbox',
-						id: checkboxId,
-						name: label,
-						value: label
-					}
-				},
-				{
-					tag: 'label',
-					attributes: {
-						for: checkboxId
-					}
-				},
-				{
-					text: label
-				}
-			]
+		checkboxView.set( {
+			isVisible: true,
+			tooltip: true,
+			label,
+			checkboxId
```

Comment:
You're proposing using fixed `id` for checkbox (it's `'matchcase'` or `'wholewords'`).

The problem is that there can be 0..n dropdowns on the same page (you can have multiple editors). This would result with checkboxes with duplicated ids.

Instead you should use `uid()` util.